storage.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	model "github.com/golang-bristol/beer-model"
+)
+
+const (
+	// StorageTypeJSON is json type indicator
+	StorageTypeJSON = "JSON" // storage_json.go
+
+	// StorageTypeMemory will store data in memory
+	StorageTypeMemory = "Memory" // storage_mem.go
+)
+
+// Storage represents all possible action available to deal with data
+type Storage interface {
+	SaveBeer(...model.Beer) error
+	SaveReview(...model.Review) error
+	FindBeer(model.Beer) ([]*model.Beer, error)
+	FindReview(model.Review) ([]*model.Review, error)
+}
```

Comment:
Think this should have FindBeers() error and FindReviews() error as well (for efficiency / simplicity reasons explained below)